sites/website/docusaurus.config.js

```diff
@@ -13,12 +13,19 @@ module.exports = {
             async: true,
         },
     ],
-    themes: ["@docusaurus/theme-live-codeblock"],
+    themes: [require.resolve("@docusaurus/theme-live-codeblock")],
     themeConfig: {
         navbar: {
             logo: {
                 alt: "FAST-DNA",
```

Comment:
For accessibility, the alt element should be more specific, "FAST logo containing ..." which visually describes what a visually impaired user may not otherwise see.